### PR TITLE
isogram: add test generator

### PR DESCRIPTION
**Pull request by ferhatelmas**
Part of #605.

### CODE REVIEW

**Comment by tleen** [comment]
Sorry for the delay I've been AFK for a few days. Will catch up on these soon.

**Comment by ferhatelmas** [comment]
@tleen No worries :smile:

File: exercises/isogram/isogram_test.go [hunk 1]
```diff
@@ -2,29 +2,13 @@ package isogram
...
-	{"Emily Jung Schwartzkopf", true},
-	{"éléphant", false},
-}
-
 func TestIsIsogram(t *testing.T) {
 	for _, c := range testCases {
-		if IsIsogram(c.word) != c.expected {
-			t.Fatalf("FAIL: Word %q, expected %v, got %v", c.word, c.expected, !c.expected)
+		if IsIsogram(c.input) != c.expected {
+			t.Fatalf("FAIL: Word %q, expected %t, got %t", c.input, c.expected, !c.expected)
```
Comment (tleen) [inline exercises/isogram/isogram_test.go]: Do you think its worth adding the test case description here as an indicator for what went wrong?

Comment (ferhatelmas) [reply exercises/isogram/isogram_test.go]: I think it's better, added.

**Pull request merged**